projects/cashmere/src/lib/banner/hc-banner.html

```diff
@@ -0,0 +1,6 @@
+<div class="hc-banner {{type}}" [ngClass]="{'hc-banner-dismiss': clickDismiss}" (click)="_bannerClick($event)">
+    <ng-content></ng-content>
+    <span *ngIf="clickDismiss" class="hc-banner-close">
+        <div [ngClass]="type != 'info' ? 'hc-banner-close-icon' : 'hc-banner-close-icon-dark'"></div>
```

Comment:
I'd consider dropping the logic in the template here, just use `hc-banner-close-icon`, and then when you need the dark close icon you could target with `.info .hc-banner-close-icon`

Reply:
great suggestion, much cleaner that way.  just pushed a commit that updates as you recommended